### ISSUE TITLE
Trying to download JDK 8.0.275.01 (x86) Windows - is there a list of release dates of versions available?

### ISSUE THREAD

**Issue by pinkie2003** [AdoptOpenJDK/openjdk-support#278]
Hi All,
Windows 10 workstation running File Catalyst v3.4.2 as part of a dental application.
Having a problem with File Catalyst throwing a java/lang/NoClassDefFoundError: java/lang/Object error when attempting to start it's service.
JAVA_HOME variable is set correctly. 
Research leads me to believe the problem is it can't open rt.jar although the file is there but not on the same path as a current install of AdoptOpenJDK.
Want to try reinstall of the original version of the software. Been randomly downloading binaries from the archive trying to find the right version but hoping there is an easier way.
Thanks for reading, hope you can help
Ray

**Comment by aahlenst** [comment]
Does https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=hotspot help? Scroll down to jdk8u275-b01.

**Comment by pinkie2003** [comment]
Hi Andreas,

Thanks for your posting

That is the archive I have been using to try and find the correct version. I don't see a way to scroll down to see older versions here, I see only the latest builds listed on that page. I have been selecting the Nightly builds option and picking days at random trying to find a build that matches or is close to the version I want. This is way I was after a release date for the versions as from it I could see when the version I want was released and go to that day to download the software.

Is this the correct way to find older versions?

Thanks

Ray

________________________________

**Comment by aahlenst** [comment]
Again, all releases are on https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=hotspot. I don't know what you see if you don't provide a screenshot or something similar.

The Windows versions 8u275:

* JDK: https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u275-b01/OpenJDK8U-jdk_x64_windows_hotspot_8u275b01.zip
* JRE: https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u275-b01/OpenJDK8U-jre_x64_windows_hotspot_8u275b01.zip

**Comment by pinkie2003** [comment]
Yup I'm a dummy, I have the version I was after. Thank you all for your assistance.
Ray
